Documentation/using-rkt-with-systemd.md

```diff
@@ -127,6 +131,7 @@ Environment=TMPDIR=/var/tmp
 ExecStartPre=/usr/bin/rkt fetch myapp.com/myapp-1.3.4
 # Start the app
 ExecStart=/usr/bin/rkt run --inherit-env --port=http:8888 myapp.com/myapp-1.3.4
+ExecStop=/usr/bin/rkt gc --mark-only
```

Comment:
Does it really work? According to the documentation, ExecStop is executed to stop the service (when the user runs `systemctl stop ...`), not after the rkt command terminates.
https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStop=

`ExecStopPost` could work instead.